ui/app/src/components/GlobalApp/styles.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2007-2021 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { createStyles, makeStyles } from '@material-ui/core/styles';
+
+export const useStyles = makeStyles((theme) =>
+  createStyles({
+    root: {
+      height: '100vh',
+      width: '100%'
+    },
+    drawerPaper: {
+      top: '0 !important',
+      padding: '16px'
```

Comment:
theme.spacing(2)

Reply:
Move the padding to the config. The panel should be like in preview, clear from these styles.